example-get-started-experiments/code/src/train.py

```diff
@@ -42,7 +41,6 @@ def train():
         valid_pct=params.train.valid_pct,
         item_tfms=Resize(params.train.img_size),
         batch_tfms=[
-            *aug_transforms(size=params.train.img_size),
```

Comment:
not clear why it was needed at all tbh, if we the Resize above cc @alex000kim

Reply:
Not really needed in this case and can be removed. In general, `aug_transforms` can do some transformations to the batch of images that can change their size (e.g. `RandomCrop`) from what was used earlier in `item_tfms`